vite.config.js

```diff
@@ -40,12 +40,20 @@ export default defineConfig(({ command }) => {
   const isDev = command === 'serve';
 
   const baseConfig = {
+    // Ensure all static assets are treated as
+    // 'in-scope' assets that can be tracked by vite
+    // this ensures any imported static assets are correctly
+    // mapped across file transformations.
+    assetsInclude: ['**/*.{svg,png,ico}'],
```

Comment:
We want `woff2` as well (QA noticed some rendering issues which are caused by fonts 404ing)

Reply:
I wrote int the description, I think we should just inlcude all files in the static directory as potential assets. it would be a marginal slow down in the (already much faster) build but probably worth it to know we never have this issue again.